opendbc/safety/safety/safety_honda.h

```diff
@@ -80,7 +81,8 @@ static void honda_rx_hook(const CANPacket_t *to_push) {
   int bus = GET_BUS(to_push);
 
   // sample speed
-  if (addr == 0x158) {
+  // 0x158 used for all suported Hondas except Integra (use 0x309 car_speed message)
+  if ((addr == 0x158) || (addr == 0x309)){
```

Comment:
The `0x309` case and its corresponding RxCheck aren't exercised in tests. I'm not sure we can depend on it anyway, so it may be irrelevant.

Reply:
will leave for the moment. Maybe we can find another message for speed